### PR TITLE
Re-arrange NFT details

### PR DESCRIPTION
**Pull request by soerenbf**
## Purpose

We want the NFT image to be displayed at the top of the NFT details page.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Review by shjortConcordium — COMMENTED** [review]
.

File: packages/browser-wallet/src/popup/shared/TokenDetails/TokenDetails.tsx [hunk 1]
```diff
@@ -86,6 +86,11 @@ function Nft({ token, balance, contractIndex, subIndex }: TokenProps) {
                 {thumbnail && <Img src={thumbnail.url} alt={name} withDefaults />}
                 {name}
             </h3>
+            {display?.url && (
```
Comment (shjortConcordium) [inline packages/browser-wallet/src/popup/shared/TokenDetails/TokenDetails.tsx:89]: There is no limit on the height of the image? Isn't that an issue?

Comment (soerenbf) [reply packages/browser-wallet/src/popup/shared/TokenDetails/TokenDetails.tsx:89]: I don't think so.. What's worse, to scroll a lot to see the full image, or to not be able to see it due to it being slimmed down to nothing due to a height limit? I prefer the scrolling option, I think? :shrug:

Comment (shjortConcordium) [reply packages/browser-wallet/src/popup/shared/TokenDetails/TokenDetails.tsx:89]: I was more thinking about the other data. I guess I just disagree with it being on top :thinking:

**Review by shjortConcordium — APPROVED** [review]

**Pull request merged**